python/arcticdb/version_store/library.py

```diff
@@ -1151,6 +1154,25 @@ def update(
         2018-01-01     400
         2018-01-03      40
         2018-01-04       4
+
+        Update will split the first and the last segment intersecting with ``data``
```

Comment:
Perhaps better to put this as a guide in our docs with more context rather than in the docstring? I don't think most users will have the context about what a `segment` is, what `read_index` does etc etc and it will all get too long to explain in the docstring